spec.html

```diff
@@ -23,12 +23,12 @@ <h1>Syntax</h1>
     ImportDeclaration :
       `import` ImportClause FromClause `;`
       `import` ModuleSpecifier `;`
-      <ins>`import` ImportClause FromClause IfClause `;`</ins>
-      <ins>`import` ModuleSpecifier IfClause `;`</ins>
-      <ins>`export` ExportFromClause FromClause IfClause `;`</ins>
+      <ins>`import` ImportClause FromClause [no LineTerminator here] AssertClause `;`</ins>
+      <ins>`import` ModuleSpecifier [no LineTerminator here] AssertClause `;`</ins>
+      <ins>`export` ExportFromClause FromClause [no LineTerminator here] AssertClause `;`</ins>
```

Comment:
Unrelated editorial error: This whole line should actually be part of ExportDeclaration, not ImportDeclaration.

Reply:
Resolved in #81, and rebased.